enhancements/samples/removed-bootstrapping.md

```diff
@@ -0,0 +1,220 @@
+
+In particular, the options available are:
+- mirror desired imagestream images (where typically there are multiple versions per imagestream) to 
+an available and functional registry; and mirror them in such a way such that the repository and image spec
+match what is used on registry.redhat.io
+- since there are many imagestreams, most likely add the imagestreams they are not concerned with to the 
+skip list in the operator config object
+- override the registry used by the samples operator in its config object
+- or if you want to punt on samples entirely, mark the operator as `Removed`
+
+### Goals
+
+1) Alleviate out of the install concerns in scenarios where samples installation will fail out of the gate
+
+2) Allow the cluster administrator time to decide what to do with samples in the cluster
+
+### Non-Goals
+
+Other usability improvements around installing samples from registries other than registry.redhat.io have 
+been identified in Jira:
+
+- regular expressions for further manipulation of the image reference in the imagestream
+- modes where failed imagestreams are automatically added to the skip list (where the administrator then
+automatically can have them added to the management set by fixing the import by mirroring the images, fixing
+the registry in question, etc.)
+- moving the samples operator out of the payload / CVO and making it an OLM operator, where the cluster
+administrator controls which imagestreams are installed when the provision the OLM operator
+
+Bootstrapping as removed is considered the meets minimum, least impacting change to the samples operator 
+that addresses the scenarios where out of the box imagestreams pointing to registry.redhat.io will fail.
+
+## Proposal
+
+With startup today, the samples operator bootstraps as `Managed`.  That boostrapping can be made conditional,
+and instead come up as `Removed` when the samples operator sees
+
+- Over a 3 minute window during initial install, the samples operator will attempt to create a TCP connection 
+to `registry.redhat.io` and if one never succeeds, will bootstrap as `Removed`, with no samples installed.
+- The clusteroperator `Reason` fields note the inability to access the terms based registry.
+
+The cluster administrator is then free to investigate why the inability to access `registry.redhat.com` exists
+and act accordingly.  Options could include
+
+- Samples are left in `Removed`
+- Images related to samples imagestreams are mirrored into an accessible registry, and then a) `samplesRegistry` is
+updated to that accessible registry in the sample operator's config object, b) Imagestreams not mirrored are added
+to the config's `skippedImagestreams` list, c) the operator is switched to `Managed`
+- Whatever proxy or underlying network condition that prevents access to `registry.redhat.io` is addressed and then
+the samples operator is switched back to `Managed`
+
+### User Stories [optional]
```

Comment:
Nit - remove optional sections that are not being used.

Reply:
removed